### PR TITLE
Small changes to `ci` tool

### PR DESCRIPTION
**Pull request by BD103**
# Objective

- Many of the items in the `ci` tool use `pub(crate)`, which is functionally equivalent to `pub` when the crate is not a library.
- A few items are missing documentation.

## Solution

- Make all `pub(crate)` items just `pub`.
  - `pub` is easier to type and less obscure, and there's not harm from this change.
- Add / modify documentation on `CI`, `Prepare`, and `PreparedCommand`.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]
1. I like the docs.
2. Slightly positive on using `pub`.
3. Neutral to slightly negative on mod.rs -> command.rs but I super don't care.

File: tools/ci/src/commands.rs
Comment (MrGVSV) [inline tools/ci/src/commands.rs:1]: This is one of those changes that feels more preference-based. There's nothing wrong with doing it this way, but I will say that a majority of the repo uses `mod.rs` (which also happens to be my preference).

I’ll suggest sticking with the project's conventions if only to maintain consistency. But it's not a big deal either way.

**Review by MrGVSV — APPROVED** [review]
Nice! Thanks for adding the docs, especially to the `Prepare`-related items.

Just one bikeshedding comment.

**Comment by BD103** [comment]
Ok, I'm think I'm going to revert the `mod.rs` change. Neither of you really liked the change, and I suppose the discoverability benefit is overshadowed by the inconsistency.

👍

**Pull request merged**